pkg/kyamls/helpers_test.go

```diff
@@ -0,0 +1,69 @@
+package kyamls
```

Comment:
can we change the it to kaymls_test. None of the tests are testing unexported functions.

Reply:
Sure.